Source/WebKit/Platform/IPC/Connection.cpp

```diff
@@ -128,23 +134,40 @@ class Connection::SyncMessageState {
     };
     Deque<ConnectionAndIncomingMessage> m_messagesBeingDispatched; // Only used on the main thread.
     Deque<ConnectionAndIncomingMessage> m_messagesToDispatchWhileWaitingForSyncReply WTF_GUARDED_BY_LOCK(m_lock);
+
+    RunLoop& m_runLoop;
+    int m_clients WTF_GUARDED_BY_LOCK(syncMessageStateMapLock) { 0 };
```

Comment:
Why are we using an int and not an unsigned? Looks a little odd.

Reply:
I think in C++ the idea is that all other aspects being equal, int is the "more optimal" in the way that it communicates to the compiler that the value  value cannot overflow.

underflow in unsigned -- if possible, it would be my error
overflow in unsigned -- if possible, it would be my error

going negative in int -- if possible, it would be my error
overflow in int -- if possible, it would be my error

I can of course change it to unsigned..